pilot/pkg/networking/core/v1alpha3/tracing.go

```diff
@@ -150,6 +151,8 @@ func configureFromProviderConfig(pushCtx *model.PushContext, meta *model.NodeMet
 		})
 
 	case *meshconfig.MeshConfig_ExtensionProvider_Skywalking:
+		rfCtx.StartChildSpan = true
```

Comment:
Mutating a pointer feels tricky to understand and refactor. Would it be simpler to just return a value from configureTracing?

Reply:
fixed